opm/autodiff/SimulatorFullyImplicitBlackoil_impl.hpp

```diff
@@ -483,13 +483,15 @@ namespace Opm
                       const WellMap&    wmap)
         {
             std::vector<int> resv_prod;
-
-            for (int w = 0, nw = wells.number_of_wells; w < nw; ++w) {
-                if (is_resv_prod(wells, w) ||
-                    ((wells.name[w] != 0) &&
-                     is_resv_prod(wmap, wells.name[w], step)))
-                {
-                    resv_prod.push_back(w);
+            if( &wells )
```

Comment:
Well, every day you learn something ;-) and that's the new thing for today. 
You can easily do 

const Wells& wells = *((Wells *) 0);

That is basically what happens when the *wells is passed to the Simulator without checking whether it's NULL or not.